erpnext_documentation/www/docs/user/manual/en/erpnext_integration/dropbox-backup.md

```diff
@@ -87,4 +87,6 @@ So for an app named `erpnext`, following are the folder locations:
 Database files: /Apps/erpnext/database
 Public files: /Apps/erpnext/files
 Private files: /Apps/erpnext/private/files
-```
\ No newline at end of file
+```
+
+> **Note**: If the compressed backup size exceeds 1GB (Gigabyte), the system will upload the latest available backup to Dropbox instead of generating a new backup file.
```

Comment:
instead of generating a new backup file in the system?

Reply:
not needed